awx/ui_next/src/components/Workflow/WorkflowActionTooltipItem.jsx

```diff
@@ -27,12 +27,14 @@ function WorkflowActionTooltipItem({
   onClick,
   onMouseEnter,
   onMouseLeave,
+  ...rest
```

Comment:
do we need to `..rest` or can we explicitly list the `ids`

Reply:
either way